source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -0,0 +1,41 @@
+=====================================
+Upgrading Dovecot from 2.3 to 2.4/3.0
+=====================================
+
+.. contents::
+   :depth: 1
+   :local:
+
+
+About version numbers
+=====================
+
+With 2.4 / 3.0 release, the CE and Pro releases have different major version numbers.
+In documentation any reference to 2.4 applies to 3.0 as well, and usually (but not always) vice versa.
+
+Upgrade path
+============
+
+Before upgrading, please look at the list of removed features carefully.
+If you are using a removed compression mechanism, you will likely need to migrate your setup.
```

Comment:
There is only one removed compression mechanism, so maybe say it specifically that xz was removed?